maintainers/scripts/debian-patches.sh

```diff
@@ -1,17 +1,17 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
I think right now our setup is to have mandatory `/bin/sh` but optional `/ur/bin/env`